Alloy/Metal.swift

```diff
@@ -75,18 +75,16 @@ public final class MTLContext {
                   library: library)
```

Comment:
Not related to line
# MTLContext
* remove device from init alongside library
* remove availability from init
* migrate to throwable 
* assign library to cache with bundle as a key
* move MTLcontext related code to separate file